adoc/headers/buffer.h

```diff
@@ -2,29 +2,6 @@
 // SPDX-License-Identifier: Apache-2.0
 
 namespace sycl {
-namespace property {
-namespace buffer {
-class use_host_ptr {
- public:
-  use_host_ptr() = default;
-};
-
-class use_mutex {
- public:
-  use_mutex(std::mutex& mutexRef);
-
-  std::mutex* get_mutex_ptr() const;
-};
-
-class context_bound {
- public:
-  context_bound(context boundContext);
-
-  context get_context() const;
-};
-} // namespace buffer
-} // namespace property
```

Comment:
Is this really to be removed? :thinking:

Reply:
This was moved per @gmlueck's [request](https://github.com/KhronosGroup/SYCL-Docs/pull/824#issuecomment-2901512286):

> Remove the properties from the top-level synopsis. You have the synopses for these properties under the "Properties" section, and there is no need to duplicate them.

Unless I misunderstood the request.

Reply:
I think we resolved this in the WG meeting after @keryell added this comment.  The synopses for the properties weren't removed.  They were just moved to a different place in the document.  I don't think anyone has concerns about that.